src/radical/pilot/bootstrapper/default_bootstrapper.sh

```diff
@@ -1414,6 +1414,22 @@ profile_event 'cleanup done'
 echo "#"
 echo "# -------------------------------------------------------------------"
 
+echo
+echo "# -------------------------------------------------------------------"
+echo "#"
+echo "# Tarring profiles ..."
+PROFILES_TARBALL="profiles.tgz"
+tar -czf $PROFILES_TARBALL *.prof
```

Comment:
Uh, `-czf`?  You live a dangerous life... ;)  Lets see if you know what I mean...  The line is correct though...